Jenkinsfile

```diff
@@ -38,6 +38,7 @@ stage("Build and Publish") {
       conda activate ${ENV_NAME}
       ./static/cache.sh restore _build/eval_pytorch/data
       d2lbook build eval --tab pytorch
+      d2lbook build slides --tab pytorch
```

Comment:
Why do we need this? The slides feature was specific to d2l-en, and I don't think we have any separate slides repos specifically for different translations at the moment.
```suggestion
```